docs/quick-tutorials/k8s-kafka-mtls.mdx

```diff
@@ -256,49 +229,67 @@ so we can see the effects of our changes in real time.
+[client] Sending messages
+[debug]  [client] Sent message 1 [transactions]
+[client] Sent message to topic: transactions - [client] Sent message 1 [transactions]
+[client] Sent message to topic: mytopic - [client] Sent message 1 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 27 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 18 [mytopic]
+[client] Read message from topic: mytopic - [client] Sent message 2 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 36 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 6 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 33 [mytopic]
+[client] Read message from topic: mytopic - [client-other] Sent message 19 [mytopic]
+```
+
+As you can see, both `client` and `client-other` are currently able to access both `mytopic` and `transactions` topics.
 
 3. **Open another terminal window [client-other]** and tail the client-other log:
 ```bash
 kubectl logs -f --tail 1 -n otterize-tutorial-kafka-mtls deploy/client-other
 ```
-This other client should also be able to communicate with the server:
+This other client should also be able to access both topics:
 ```
-Loading mTLS certificates
-Connecting to Kafka
-Creating a producer for - mytopic
-Sending messages
-Sent message - Message 1 [sent by client-other]
-Sent message - Message 2 [sent by client-other]
-Sent message - Message 3 [sent by client-other]
+[client-other] Loading mTLS certificates
+[client-other] Connecting to Kafka
+[client-other] Creating a producer and a consumer for - mytopic
+[client-other] Sending messages
+[client-other] Creating a producer and a consumer for - transactions
+[client-other] Sending messages
+[debug]  [client-other] Sent message 1 [mytopic]
+[debug]  [client-other] Sent message 1 [transactions]
+[client-other] Sent message to topic: transactions - [client-other] Sent message 1 [transactions]
+[client-other] Sent message to topic: mytopic - [client-other] Sent message 1 [mytopic]
+[client-other] Read message from topic: transactions - [client] Sent message 5 [transactions]
 ```
 
 If you've attached Otterize OSS to Otterize Cloud, you can now browse to your account at [https://app.otterize.com](https://app.otterize.com) and see the access graph for your cluster:
 
 ![Access graph](/img/quick-tutorials/k8s-kafka-mtls/base.png)
 
+You can see from the Access Graph that no clients are currently blocked because we haven't enabled any sort of enforcement yet.
+
 <details>
-<summary>Why do I see six services?</summary>
+<summary>Why do I see eight services?</summary>
+
+In addition to the Kafka service and the 3 clients we deployed (yes, we snuck a 3rd client in there), the network mapper also picked up the calls between the intents operator and Kafka, and between Kafka and zookeeper, so those discovered intents are reflected in the access graph (in light blue).
```

Comment:
I think we may no longer need the 3rd client, so how about we remove it and then we can remove this comment?

Reply:
Removed that reference and decremented the number of services we see.